inspector/an-inspector/ForgeModule/src/io/trigger/forge/android/modules/tabs/ModalView.java

```diff
@@ -456,13 +454,23 @@ private TextView createTitle(DisplayMetrics metrics, String title, JsonArray tit
             titleColor = Color.argb(titleTint.get(3).getAsInt(), titleTint.get(0).getAsInt(), titleTint.get(1).getAsInt(), titleTint.get(2).getAsInt());
         }
         titleView.setTextColor(titleColor);
-        titleView.setTextSize(TypedValue.COMPLEX_UNIT_PX, metrics.density * 24);
-        titleView.setGravity(Gravity.LEFT);
+        titleView.setGravity(Gravity.CENTER_VERTICAL);
```

Comment:
I tested on one Android device (HTC Android 8.0.0) and things looked good: left aligned & vertical centered.

Still, I'm unsure if the gravity should be like this:

```suggestion
        titleView.setGravity(Gravity.LEFT | Gravity.CENTER_VERTICAL);
```

🤷🏻‍♂️

Reply:
We stick with the current setting, since left seems to be the default and is inherited.